addons/udes_sale_stock/tests/common.py

```diff
@@ -0,0 +1,38 @@
+from odoo.addons.udes_stock.tests.common import BaseUDES
+
+
+class BaseSaleUDES(BaseUDES):
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Creating a BaseSaleUDES testing class on top of BaseUDES which we can reuse when we need to have BaseUDES with
```

Comment:
this docstring can live on the class and the `setUpClass(cls):` function can be removed